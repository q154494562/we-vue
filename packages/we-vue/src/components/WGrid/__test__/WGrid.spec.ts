import { shallowMount } from '@vue/test-utils'
import Grid from '../'

describe('grid', () => {
  test('create', () => {
    const wrapper = shallowMount(Grid, {
      propsData: {},
    })

    expect(wrapper.classes()).toContain('weui-grids')

    expect(wrapper.html()).toMatchSnapshot()
  })
})
