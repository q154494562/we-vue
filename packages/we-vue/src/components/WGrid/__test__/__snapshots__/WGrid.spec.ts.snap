// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grid create 1`] = `<div class="weui-grids"></div>`;
